packages/terra-application-docs/src/terra-dev-site/tool/terra-dev-site/UpgradeGuides.f/v7.0.0.s.tool.mdx

```diff
@@ -0,0 +1,150 @@
+|`hotReloading`|**removed**|True in dev false in prod|
+|`npmPackage`|**removed**|Only the root npm package is used.|
+|`sideEffectImports`|`sideEffectImportFilePaths`|The name now includes 'filePaths' to better describe the expected input.|
+|`placeholderSrc`|**removed**|No longer used.|
+|`readMeContent`|`primaryNavigationItems.additionalContent`|No longer a one off, consumers can now add additional content to each primary navigation item that would not normally be found by searching.|
+|`appConfig.title`|`titleConfig.title`|Moved to a titleConfig object to align with terra-application-navigation config.|
+|`appConfig.favicon`|`faviconFilePath`|No longer nested under appConfig. The name now includes 'filePaths' to better describe the expected input.|
+|`appConfig.themes`|**removed**|The available themes are now pulled from the terra-theme.config file via the webpack global variable `TERRA_THEME_CONFIG`.|
+|`appConfig.defaultTheme`|`defaultTheme`|No longer nested under appConfig.|
+|`appConfig.locales`|**removed**|Deprecated as of terra-dev-site 6.23, now removed.|
+|`appConfig.defaultLocale`|`defaultLocale`|No longer nested under appConfig.|
+|`appConfig.defaultDirection`|`defaultDirection`|No longer nested under appConfig.|
+|`appConfig.extensions`|`extensionItems`|Renamed to align with terra-application-navigation config.|
+|`appConfig.extensions.iconPath`|`extensionItems.iconPath`|Renamed to align with terra-application-navigation config.|
+|`appConfig.extensions.key`|`extensionItems.key`|Renamed to align with terra-application-navigation config.|
+|`appConfig.extensions.text`|`extensionItems.text`|Renamed to align with terra-application-navigation config.|
+|`appConfig.extensions.componentPath`|`extensionItems.modalFilePath`| Renamed to prepare for application v2 modals.|
+|`appConfig.extensions.size`|**removed**|One size fits all.|
+|`appConfig.headHtml`|`headHtml`|No longer nested under appConfig.|
+|`includeTestEvidence`|**removed**|Removed the ability to include test evidence, this should be covered now by a separate site report.|
+
+## Dependency changes
+
+Previously `terra-dev-site` was providing a version of `rect-intl` as a dependency. Because this satisfied the `terra-application` peer dependency requirement, many applications did not define it. With this release we have removed that dependency and applications will have to provide that peer dependency themselves.
+
+`terra-application` is now a peer dependency and must be defined by the package consuming `terra-dev-site`.
+
+`terra-toolkit` is no longer a peer dependency to allow consumers to use `terra-dev-site` v7 with either terra-toolkit or the replacement `terra-functional-testing` package.
+
+### Resulting Test changes
+
+The page introduces a new tab stop. Any tests using tabstops will have to be updated to include an additional tab stop.
```

Comment:
Do you have an example of what you mean?

Reply:
Honestly I don't think this is still true, I'll remove it.